Задание8.3/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace Задание8._3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var n = int.Parse(Console.ReadLine());
+            var m = 0;
+            while ((m * m * m + m + 1) < n)
+                m++;
+            Console.WriteLine(m * m * m + m + 1);
```

Comment:
Чтобы два раза не вычислять
```suggestion
            var m = 0;
            int k;
            while ((k = m * m * m + m + 1) < n)
                m++;
            Console.WriteLine(k);
```